grpc/samples/greeter/Makefile

```diff
@@ -4,11 +4,11 @@ LDFLAGS ?=
 .PHONY: all
 all: server client
 
-greeter_generated.h: greeter.fbs
```

Comment:
I don't understand this change.. surely you'd want the `.h` in there?

Reply:
Perhaps it's the way i am using make?

When simply running `make` the `server` target is run, and does not find a `greeter_generated.h` file. I changed this and now at first, the `greeter_generated` target is run for the `client` & `server` ones.